docs/swarmmode.md

```diff
@@ -0,0 +1,73 @@
+# Microsoft Azure Container Service Engine - Swarm Mode Walkthrough

```

Comment:
note, this is just scoped to the docs.  The references in the code to swarm mode are fine.